includes/admin/views/html-admin-page-addons.php

```diff
@@ -10,9 +10,14 @@
 }
 
 echo '<h1 class="screen-reader-text">' . esc_html__( 'WP Job Manager Add-ons', 'wp-job-manager' ) . '</h1>';
+echo '<div class="wpjm-extensions-filter-search">';
 if ( ! empty( $categories ) ) {
 	// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Input is used safely.
 	$current_category = isset( $_GET['category'] ) ? sanitize_text_field( wp_unslash( $_GET['category'] ) ) : '_all';
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Input is used safely.
+	if ( isset( $_GET['search'] ) && ! empty( $_GET['search'] ) ) {
+		$current_category = null;
```

Comment:
My thought with this was that if I see the `All` tab highlighted, I would assume that I am seeing ALL the extensions - not just the search results.  I can see what you mean though, where if you were in a specific category and searched you might think that you're searching within that category.  However, do you think by NOT selecting a tab it might give the user a visual clue that they are not in that category any more?

Curious what @yscik and @fjorgemota think.

Reply:
Yeah since we are not searching within the category it's probably better to just deselect it and not select 'All' either.